pycontw/pycontw-2021
# add auth header in request

## Types of Changes

**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies.


## Description

add auth header in request

## Steps to Test This Pull Request

1. set var `AUTH_TOKEN` in env
2. start service check request include auh header
